packages/services/api/src/modules/audit-logs/module.graphql.mappers.ts

```diff
@@ -0,0 +1,21 @@
+import { AuditLogEvent } from './providers/audit-logs-types';
+
+export type AuditLogMapper = AuditLogEvent;
```

Comment:
I think you can just re-export this one, and point all types in codegen config file to the same type?